pages/index.js

```diff
@@ -228,7 +229,7 @@ export default function Home({
                       onClickParent();
                       setSelectedChild(null);
                       setShowAddAccount(false);
-                      setAccount("");
+                      handleAccount("");
                       handleClickLink("home");
```

Comment:
I am not sure what `handleAccount` does, but later on it's called with an account object. Here you call it with an empty string - what is it supposed to do with an empty string? If you want to reset the account to something like "not selected", consider to use `null` instead.

Reply:
Thank you for the feedback, I have changed it accordingly. Thanks.